### PR TITLE
Fix mobile blob font, line height and grid auto row with media queries

### LINKED ISSUE #96: Fix font & line height on mobile
**Is your feature request related to a problem? Please describe.**
Some ingredient names go outside of blob on mobile(too long).

**Describe the solution you'd like**
Fix line height or size.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Added media queries to decrease line height and font size on some mobile devices.
2. Decrease `grid-auto-row` in `BlobsContainer` to `175px` to decrease gap on mobile.

## Purpose
Mobile sizing.

## Approach
Media queries.

Closes #96

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Nice!

**Review by BlackCubes — APPROVED** [review]
Good job!

File: src/components/Blob.tsx [hunk 1]
```diff
@@ -59,6 +59,15 @@ const IngredientName = styled.h1`
     font-size: 20px;
     line-height: 28px;
   }
+  @media (max-width: 375px), and (max-height: 812px) {
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  @media (max-width: 320px), and (max-height: 568px) {
```
Comment (dannysanchez559) [inline src/components/Blob.tsx:67]: iPhone SE screen size. Nice :)

**Review by dannysanchez559 — COMMENTED** [review]
looks good!

**Pull request merged**